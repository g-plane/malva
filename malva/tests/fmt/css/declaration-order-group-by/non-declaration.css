/*cfg declarationOrder = "alphabetical"
      declarationOrderGroupBy = "nonDeclaration"*/
div {
  z: 0;
  y: 0;
  x: 0;
  & {}
  d: 0;
  c: 0;

  b: 0;
  a: 0;
}
