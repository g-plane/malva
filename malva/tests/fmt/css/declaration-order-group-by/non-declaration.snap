---
source: malva/tests/fmt.rs
---
/*cfg declarationOrder = "alphabetical"
      declarationOrderGroupBy = "nonDeclaration"*/
div {
  x: 0;
  y: 0;
  z: 0;
  & {}
  a: 0;
  b: 0;
  c: 0;
  d: 0;
}
